src/main/java/frc/robot/constants/VisionConstants.java

```diff
@@ -39,8 +40,8 @@ public static void update() {
         new Pair<String, Transform3d>(
           "Camera_2",
           new Transform3d(
-            new Translation3d(-Units.inchesToMeters(4.75), Units.inchesToMeters(10.375), Units.inchesToMeters(10)),
-            new Rotation3d(0, 0, 0)
+            new Translation3d(Units.inchesToMeters(15), Units.inchesToMeters(-3.3125), Units.inchesToMeters(10)),
+            new Rotation3d(0, 0, Math.PI)
```

Comment:
why is the rotation PI, the camera on the test bot is faceing the same direciton as the robot.

Reply:
This is what worked on the classbot. 0 made the robot's rotation 0 when it faced toward red, and when I moved forward, it's pose moved in the negative x direction instead of positive x. This fixed it. π is facing toward red and 0 is facing toward blue.